jenkins/hybrid_execution.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+. jenkins/version-def.sh
+
+export GLUTEN_BUNDLE_JAR
+export HYBRID_JAR
+export GLUTEN_THIRDPARTY_JAR
+
+hybrid_prepare(){
+    echo "Checking hybrid exeicution tests environmet..."
+    local cup_arch=$(uname -m)
+    local os_version=$(source /etc/os-release > /dev/null 2>&1 && echo ${ID}_${VERSION_ID})
+    echo "cup_arch=$cup_arch, os_version=$os_version, SCALA_BINARY_VER=$SCALA_BINARY_VER"
+    if [[ ! ("$cup_arch" == "x86_64" && ("$os_version" == "ubuntu_20.04" || "$os_version" == "ubuntu_22.04") && "$SCALA_BINARY_VER" == "2.12") ]]; then
+        echo "SKIP! Only supports running Scala 2.12 hybrid execution tests on an x86_64 processor under Ubuntu 20.04 or 22.04."
```

Comment:
return early?

Reply:
Add `return 1` to exit early

Reply:
fixed by https://github.com/NVIDIA/spark-rapids/pull/11906/commits/9700b4078af8f496e93ef9cb26472e4dd79bae31